### ISSUE TITLE
Readonly `NativeSelect` missing kb focus

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#1659]
Reported by community in DM on Slack.

`NativeSelect` is not possible to focus using keyboard like our other forms components.

`NativeSelect` has a `readonly` prop which applies readonly-styling and disables the field because [select does not support readonly ](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#overview). 

This breaks content flow in form with keyboard when using it with other elements that are readonly. 

We should investigate if we should mimic readonly state for our `NativeSelect`, looking at other libraries it seems to be a common thing to do. 

Note:
- We are already mimicing readonly state for our `Radio` and `Checkbox` which does also not natively support `readonly` state.

**Comment by mimarz** [comment]
By vote on Slack we will do this change
